internal/quickstart/choose_sdk.go

```diff
@@ -32,15 +33,17 @@ type chooseSDKModel struct {
 }
 
 func NewChooseSDKModel(selectedIndex int) tea.Model {
+	initSDKs()
 	l := list.New(sdksToItems(), sdkDelegate{}, 30, 9)
-	l.Title = "Select your SDK:\n"
+	l.FilterInput.PromptStyle = lipgloss.NewStyle()
+
+	l.Title = "Select your SDK:"
```

Comment:
Removed the newline and added a margin bottom to work with the title text and the filter title text.